plugins/inventory/windows_azure.py

```diff
@@ -163,11 +203,14 @@ def add_cloud_services(self):
             sys.exit(1)
 
     def add_deployments(self, cloud_service):
-        """Makes an Azure API call to get the list of virtual machines associated with a cloud service"""
+        """Makes an Azure API call to get the list of virtual machines
+        associated with a cloud service.
+        """
         try:
             for deployment in self.sms.get_hosted_service_properties(cloud_service.service_name,embed_detail=True).deployments.deployments:
-                if deployment.deployment_slot == "Production":
-                    self.add_deployment(cloud_service, deployment)
+                self.add_deployment(cloud_service, deployment)
+                #if deployment.deployment_slot == "Production":
+                #    self.add_deployment(cloud_service, deployment)
```

Comment:
Removed for now.  It appears it was changed in Tower to return all hosts instead of just "Production".  Might be nice to have as a config option, but so far I haven't seen any requests for that feature.